spec.md

```diff
@@ -2334,6 +2334,52 @@ The width of each primitive operation is detailed in [@sec:primitive-operations]
 The width of the integer literal expressions is detailed in their respective
 sections.
 
+# Combinational Loops
+
+Combinational logic is a section of logic with no registers between gates.
+A combinational loop exists when the output of some combinational logic 
+is fed back into the input of that combinational logic with no intervening
+register. FIRRTL does not support combinational loops even if it is possible
+to show that the loop does not exist under actual mux select values.
```

Comment:
Perhaps, more directly, "FIRRTL does not allow combinatorial loops.  The existence of a combinatorial loop is a static, structural property of the code, not dependent on runtime-values."